_build/partials/interactive-map.hbs

```diff
@@ -1,31 +1,45 @@
 <div id="leaflet-map"></div>
 <script>
 
-    function copyGeojsonOnClick(geometry){
+    function copyGeojsonOnClick(geometry) {
         navigator.clipboard.writeText(JSON.stringify(geometry));
     }
 
-    async function loadData(){
+    async function loadData() {
         const polygonsResponse = await fetch("polygons.geojson");
         const geojsonPolygons = polygonsResponse.status === 404 ? null : await polygonsResponse.json();
 
         return geojsonPolygons;
     }
 
-    function generatePopupText(feature){
+    function generatePopupText(feature, centroid) {
         // replace " with ' so the string can be passed as a function param in HTML
         const encodedString = JSON.stringify(feature.geometry).replace(/"/g, "'");
 
-        return "<h4>" + feature.properties.geolocation +"</h4>" +
+        let eobUrl = new URL("https://apps.sentinel-hub.com/eo-browser/");
+        eobUrl.search = new URLSearchParams({
+            themeId: "PLANET_SANDBOX",
+            lat: centroid.lat,
```

Comment:
Would adding the parameter `zoom: 14` fix the problem that Dino mentioned? I think most of the tiles should be properly visible on that zoom level.

Reply:
It would. Added the zoom.

Reply:
I think it looks good now 👍